### PR TITLE
Feat: Manage department

### PR DESCRIPTION
**Pull request by rarumdj**

### CODE REVIEW

File: src/components/layout/skeleton.tsx [hunk 1]
```diff
@@ -130,3 +131,48 @@ export const ProfileSkeletonMini: React.FC<{
...
+//     <Text
+//         ml="12"
+//         flexWrap="wrap"
+//         fontWeight="400"
+//         _dark={{ color: 'gray.400' }}
+//         _light={{ color: 'gray.600' }}
+//     >
+//         {item.value}
+//     </Text>
+// </Stack>
```
Comment (chukwuma-azubuike) [inline src/components/layout/skeleton.tsx:178]: You can as well delete this.

File: src/store/services/account.ts [hunk 2]
```diff
@@ -248,6 +258,7 @@ export const accountServiceSlice = createApi({
 export const {
     useSendOTPQuery,
     useGetUsersQuery,
+    useGetCampusSummeryByCampusIdQuery,
```
Comment (chukwuma-azubuike) [inline src/store/services/account.ts:261]: Typographical error `useGetCampusSummeryByCampusIdQuery`, supposed to be `useGetCampusSummaryByCampusIdQuery`

File: src/views/app/Workforce-management/global-workforce.tsx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    const {
+        user: { department, campus },
+        isHOD,
+        isAHOD,
+        isCampusPastor,
+        isGlobalPastor,
+        isQC,
+    } = useRole();
+
+    const { data, isLoading, isSuccess, isFetching } = useGetCampusSummeryByCampusIdQuery(campus._id);
```
Comment (chukwuma-azubuike) [inline src/views/app/Workforce-management/global-workforce.tsx:29]: Typo fix as well

File: src/views/app/Workforce-management/global-workforce.tsx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            color: 'red.700',
+        },
+    ];
+
+    const Departmentlist = data?.departmentCount.map((item, index) => ({
+        ...item,
+        title: item.departmentName,
+        value: item.userCount,
+        flex: index % 2 === 0 ? 1 : 0, // set flex to 1 for even index and 0 for odd index
+    }));
```
Comment (chukwuma-azubuike) [inline src/views/app/Workforce-management/global-workforce.tsx:70]: You should memoize this for better performance.

File: src/views/app/Workforce-management/global-workforce.tsx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                                            <Heading size="md" _dark={{ color: 'white' }} _light={{ color: 'black' }}>
+                                                {item.value}
+                                            </Heading>
+                                        </Stack>
+                                    </TouchableOpacity>
+                                </VStack>
+                            </Stack>
+                        ))}
+                    </Stack>
+                </Center>
```
Comment (chukwuma-azubuike) [inline src/views/app/Workforce-management/global-workforce.tsx:185]: This should be a separate component for reusability.

File: src/views/app/Workforce-management/workforce-management.tsx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+const WorkforceManagement: React.FC<NativeStackScreenProps<ParamListBase>> = ({ navigation, route }) => {
+    const campusUsersData = route?.params as IcampusUserData['departmentCount'][0];
+    const { isCampusPastor, isQC, isGlobalPastor, isSuperAdmin } = useRole();
+
+    const { setOptions } = navigation;
+    const isScreenFocused = useIsFocused();
+
+    useFocusEffect(
+        React.useCallback(() => {
+            setOptions({ title: `${campusUsersData.departmentName || 'Workforce Management'}` });
```
Comment (chukwuma-azubuike) [inline src/views/app/Workforce-management/workforce-management.tsx:28]: There is already a hook for this. Check `useScreenFocus`

**Review by chukwuma-azubuike — CHANGES_REQUESTED** [review]
Please include screenshots and confirm that the issue with the `goBack` navigation is working.

**Comment by rarumdj** [comment]
the back button is not working because of the implementation you did on the index file please help check it @chukwuma-azubuike

**Review by chukwuma-azubuike — APPROVED** [review]
LGTM

**Pull request merged**